src/components/ContactsTabContent/ContactsTabContent.tsx

```diff
@@ -12,21 +12,20 @@ import {
   contactsColumns,
   DataTable,
   NoRecordsText,
-  TransactionEditDialogProps,
 } from '@/components';
 import { ContactExtended } from '@/interfaces/contacts.ts';
 
 export interface ContactsTabContentProps {
   contacts: ContactExtended[];
-  EditDialog?: React.ComponentType<ContactEditDialogProps | TransactionEditDialogProps>;
+  ContactEditDialog?: React.ComponentType<ContactEditDialogProps>;
```

Comment:
Is this change is related with changing `any` types to specific types? 🤔

Reply:
Yes

Reply:
how?

Reply:
The `any` issue was deeper in the code. This change is needed to handle `any` issue